corehq/apps/analytics/templates/analytics/forms/bootstrap3/hubspot_cta_form.html

```diff
@@ -58,79 +47,77 @@
                class="form-control">
       </div>
     </div>
-    <div class="form-group"
-         data-bind="visible: showContactMethod">
-      <label class="control-label">
-          {% trans "How should we contact you?" %}<span class="asteriskField">*</span>
-      </label>
-      <div class="controls">
-        <select name="preferred_method_of_contact"
-                data-bind="value: preferred_method_of_contact"
-                class="select form-control">
-          <option value="">{% trans "Please Select" %}</option>
-          <option value="Phone">{% trans "Phone" %}</option>
-          <option value="Skype">{% trans "Skype" %}</option>
-          <!-- ko if: useWhatsApp -->
-            <option value="WhatsApp">{% trans "WhatsApp" %}</option>
-          <!-- /ko -->
-          <!-- ko if: useGoogleHangouts -->
-            <option value="Google hangouts">{% trans "Google hangouts" %}</option>
-          <!-- /ko -->
-        </select>
+    <div class="row">
+      <div class="col-sm-6">
+        <div class="form-group">
+          <label class="control-label">
+            {% trans "Organization" %}<span class="asteriskField">*</span>
+          </label>
+          <div class="controls">
+            <input type="text"
+                  name="company"
+                  data-bind="textInput: company"
+                  class="form-control">
+          </div>
+        </div>
+      </div>
+      <div class="col-sm-6">
+        <div class="form-group">
+          <label class="control-label">
+            {% trans "Job title" %}<span class="asteriskField">*</span>
+          </label>
+          <div class="controls">
+            <input type="text"
+                  name="company"
+                  data-bind="textInput: jobtitle"
+                  class="form-control">
```

Comment:
https://www.commcarehq.org/styleguide/b5/html/
```suggestion
            <input
                class="form-control"
                type="text"
                name="company"
                data-bind="textInput: jobtitle"
            />
```